src/constants/events.ts

```diff
@@ -3,3 +3,4 @@ export const ERROR_EVENT = 'error'
 export const CLOSE_EVENT = 'close'
 export const ACCOUNTS_CHANGED = 'accountsChanged'
 export const CHAIN_CHANGED = 'chainChanged'
+export const DECRYPT_ERROR = 'DECRYPT_ERROR'
```

Comment:
Don't we have to place this in another file? It is not an event...

Reply:
I will move to the index.ts file in the constants folder because that is where the two other random constants are for the access tokens. However, with the security review these are probably going away. The only other constant file is for CSS classnames which would not be appropriate.